packages/@dataform/cli/BUILD

```diff
@@ -82,21 +80,21 @@ pkg_bundle(
     ],
 )
 
-pkg_bundle(
-    name = "worker_bundle",
-    args = ["--banner='#!/usr/bin/env node'"],
-    entry_point = "worker.ts",
-    externals = externals,
-    deps = [
-        ":cli",
-    ],
-)
-
 pkg_npm_tar(
     name = "package",
     deps = [
+        "copy_bundle",
         ":bundle",
         ":package.json",
-        ":worker_bundle",
     ],
 )
+
+# Worker bundle needs to be copied over for packaging, as deps files are only
+# included if they're in this folder.
+genrule(
+    name = "copy_bundle",
+    srcs = ["//sandbox/vm:worker_bundle"],
+    outs = ["worker_bundle"],
```

Comment:
Hmmm, the output folder you pasted before - should this not be `outs = ["worker_bundle.js"]` ? I know it's executable, but it's a bit more clear if it's a JS file.

Reply:
I'm actually surprised the CLI tests pass without a `.js` extension on the file.

Reply:
Ah nice catch. I'm getting rid of the `#!/usr/bin/env node` prepended too, because that's not necessarily where the sandbox should be executing it from